tik/tikSwap/tikSwap.cpp

```diff
@@ -307,14 +417,14 @@ int main(int argc, char *argv[])
             f.open(OutputFile, std::ios::out);
             std::ostream rawStream(&f);
             raw_os_ostream raw_stream(rawStream);
-            WriteBitcodeToFile(*base, raw_stream);
+            WriteBitcodeToFile(*sourceBitcode, raw_stream);
         }
-        spdlog::info("Successfully wrote tik to file");
+        spdlog::info("Successfully wrote tikSwap to file");
```

Comment:
Same on the print. I suspect this will see increased use in more tools. Thoughts?

Reply:
Yeah, that's a good idea

Reply:
Let me know once its in.